astroid/bases.py

```diff
@@ -283,6 +283,23 @@ def callable(self):
         except exceptions.AttributeInferenceError:
             return False
 
+    def getitem(self, index, context=None):
```

Comment:
Why not put it at its original position (L319)?

Reply:
Just realized the function already existed after moving it :) Fixed.